### PR TITLE
W-16347923-deploy-json-ref-duke

### PR DESCRIPTION
**Pull request by dukesphere**

### CODE REVIEW

File: modules/ROOT/pages/ref-deployment.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  "runtime": "4.7-e-java17",
+  "workerSize": 0.1,
+  "applicationName": "my-integration-app",
+  "workers": 1,
+  "autoStart": true
+}
+----
+
+* `runtime`: Provides the Mule runtime version used by the project. 
+* `workersize`:
```
Comment (dukesphere) [inline modules/ROOT/pages/ref-deployment.adoc]: todo

File: modules/ROOT/pages/ref-deployment.adoc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "workers": 1,
+  "autoStart": true
+}
+----
+
+* `runtime`: Provides the Mule runtime version used by the project. 
+* `workersize`: 
+* `applicationName`: Name of the Mule application.
+* `workers`: 
+* `autoStart`:
```
Comment (dukesphere) [inline modules/ROOT/pages/ref-deployment.adoc]: todos

File: modules/ROOT/pages/_partials/acb-deployment.adoc [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+| `runtime`
+| Provides the Mule runtime version used by the project.  
+| CloudHub
+
+| `workersize`
+| Sets the worker memory relative to a single worker or a number of available workers. The default is `0.1`. For more information, see xref:cloudhub::cloudhub-architecture.adoc#cloudhub-workers[CloudHub Workers] in the CloudHub documentation.  
+| CloudHub
+
+| `applicationName`
+| Provides the hyphenated, lower-case name of the Mule application.
```
Comment (hannanelson) [inline modules/ROOT/pages/_partials/acb-deployment.adoc]:
```suggestion
| Provides the hyphenated, lowercase name of the Mule application.  
```

File: modules/ROOT/pages/int-deploy-mule-apps.adoc [hunk 4]
```diff
@@ -55,6 +55,19 @@ The domain name of the application varies, but it follows a similar structure su
...
 displayed in Runtime Manager, for example, `+http://app-with-100-schedulers.us-w1.cloudhub.io/api/flights+`.
 
+
+== Change a Value in deploy.json
+
+//shared 
+include::anypoint-code-builder::partial$acb-deployment.adoc[tags="deploy-json-edit"]
+
+
+== Reference
```
Comment (hannanelson) [inline modules/ROOT/pages/int-deploy-mule-apps.adoc]: Any way to make this heading more specific (like what kind of reference)?

**Review by hannanelson — DISMISSED** [review]

File: modules/ROOT/pages/int-deploy-mule-apps.adoc (hunk 4 above)
Comment (dukesphere) [reply modules/ROOT/pages/int-deploy-mule-apps.adoc]: I didn't want to use `deploy.json` as the first term in the heading. The first sentence makes it clear though, as does scanning the page, I think.

Comment (hannanelson) [reply modules/ROOT/pages/int-deploy-mule-apps.adoc]: OK. I'm just thinking for SEO and AI consumption - headings like this are verboten.

**Review by hannanelson — DISMISSED** [review]

File: modules/ROOT/pages/int-deploy-mule-apps.adoc (hunk 4 above)
Comment (dukesphere) [inline modules/ROOT/pages/int-deploy-mule-apps.adoc]:
```suggestion
== Deployment Properties 
```

**Review by hannanelson — APPROVED** [review]

File: modules/ROOT/pages/int-deploy-mule-apps.adoc (hunk 4 above)
Comment (dukesphere) [reply modules/ROOT/pages/int-deploy-mule-apps.adoc]: revised

**Review by dhairyashah-sf — APPROVED** [review]

**Pull request merged**